src/Scouting.elm

```diff
@@ -189,10 +198,7 @@ view2 model =
                 , "\nstation - "
                 , model.driverStation
                 , "\n\n"
-                , "match1 = [ 1690, 1574, 3339 || 254, 2056, 1323 ]\n"
-                , "match2 = [ 118,  1577, 1024 || 2056, 1690, 254 ]\n"
-                , "match3 = [ 1574, 3339, 1577 || 1323, 1024, 118 ]\n"
-                , "match4 = [ 3339, 1574, 1577 || 1024, 118, 2056 ]"
+                , Matches.asComment

```

Comment:
Are you sure having this hard coded here makes sense?

Reply:
It's just for me to make sure that it's displaying the correct driver station